repo/packages/M/minio/6/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "Minio",
+  "version": "0.0.7-RELEASE.2017-05-05T01-14-51Z",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/minio/minio.git",
+  "maintainer": "dev@minio.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server for cloud native applications. To deploy at scale, use DC/OS to orchestrate Minio containers one per tenant.",
+  "tags": [
+    "data",
+    "storage",
+    "filesystem"
+  ],
+  "preInstallNotes": "There is no limit to number of Minio services you can launch.",
```

Comment:
@ryadav, we wish to remove the warning about this package being preview and not production ready. Minio by itself is production ready, is there any criteria for a universe package to meet before calling itself production ready?

I will add example and a link to it once the example PR merges. Does that make sense?